wremnants/combine_theory_helper.py

```diff
@@ -119,60 +122,25 @@ def add_minnlo_scale_uncertainty(self, sample_group, use_hel_hist=False, rebin_p
         # assuming the name is unchanged
         obs = self.card_tool.fit_axes
         pt_ax = "ptVgen" if "ptVgen" not in obs else "ptVgenAlt"
-        charge_ax = "chargeVgen" if "chargeVgen" not in obs else "chargeVgenAlt"
-        sum_axes = [pt_ax, charge_ax]
 
-        syst_axes = sum_axes + ["muRfact", "muFfact"]
-        syst_ax_labels = ["PtV", "genQ", "muR", "muF"]
-        format_with_values = ["low", "center", "center", "center"]
-        if use_hel_hist:
-            syst_axes.insert(2, "helicity")
-            syst_ax_labels.insert(2, "AngCoeff")
-            format_with_values.insert(2, "low")
+        syst_axes = [pt_ax, "vars"]
+        syst_ax_labels = ["PtV", "var"]
+        format_with_values = ["edges", "center"]
 
         group_name = f"QCDscale{self.sample_label(sample_group)}"
-        # Exclude all combinations where muR = muF = 1 (nominal) or where
-        # they are extreme values (ratio = 4 or 1/4)
-        skip_entries = [{"muRfact" : 1.j, "muFfact" :  1.j}, {"muRfact" : 0.5j, "muFfact" : 2.j}, 
-                        {"muRfact" : 2.j, "muFfact" : 0.5j}]
-        # In order to make prettier names than the automated ones.
-        # No harm in leaving extra replaces that won't be triggered
-        name_replace = [("muR2muF2", "muRmuFUp"), ("muR0p5muF0p5", "muRmuFDown"), ("muR2muF1", "muRUp"), 
-                            ("muR0p5muF1", "muRDown"), ("muR1muF0p5", "muFDown"), ("muR1muF2", "muFUp"),
-        ]
+        base_name = f"{group_name}{extra_name}"
+
+        skip_entries = []
         action_map = {}
 
         # NOTE: The map needs to be keyed on the base procs not the group names, which is
         # admittedly a bit nasty
         expanded_samples = self.card_tool.getProcNames([sample_group])
         logger.debug(f"using {scale_hist} histogram for QCD scale systematics")
         logger.debug(f"expanded_samples: {expanded_samples}")
-        func = syst_tools.scale_helicity_hist_to_variations 
```

Comment:
I guess I should try the code to better see how the new version works, but could you expand on what is changing in the logic, concerning inputs and operations happening in this function? Just a few words to comment on these changes for the records (no need to leave comments in the code itself)

Reply:
The new version ```syst_tools.hist_to_variations``` actually already existed and was already used for the non-perturbative uncertainties, so this is mostly unifying/simplifying the logic.

The old function can be removed as part of a later cleanup.